spec/paths/reports@subscription-cancellation.yaml

```diff
@@ -0,0 +1,54 @@
+get:
+  tags:
+    - Reports
+  summary: Retrieve transactions with their plan data
+  description: Retrieve transactions with their plan data
+  parameters:
+    - name: periodStart
+      in: query
+      description: Report start date
+      type: string
+      required: true
+    - name: periodEnd
+      in: query
+      description: Report end date
+      type: string
+      required: true
+    - name: aggregationField
+      in: query
+      description: Report aggregation field
+      required: true
+      type: string
+      enum:
+        - canceled-by
+        - cancel-category
+        - leads.affiliate
```

Comment:
This one is more like "field name", not a "possible value". So it would be inconsistent to group by `cancel-categoty` and at the same time filter by `cancelCategory`. Do you agree?